src/utils/inputFields.json

```diff
@@ -1,4 +1,45 @@
 {
```

Comment:
What is the use case of this file?

Reply:
This file is used for defining input fields for a form